proposals/0030-dxil-vectors.md

```diff
@@ -132,6 +132,34 @@ The scalar variants of these DXIL intrinsics will remain unchanged and can be us
 This means that the same language-level vector (of any length) could be used
  in scalarized operations and native vector operations even within the same shader.
 
+### New DXIL Intrinsics
+
+#### Boolean Vector Reduction Intrinsics
+
+**VectorReduceAnd**
+
+Bitwise AND reduction of the vector returning a scalar. Return type matches vector element type.
+
+```C++
+DXIL::OpCode::VectorReduceAnd = 309
+```
+
+```asm
+ [TYPE] @dx.op.unary.v[NUM][TY](309, <[NUM] x [TYPE]> operand)
+```
+
+**VectorReduceOr**
+
+Bitwise OR reduction of the vector returning a scalar. Return type matches vector element type.
+
+```C++
+DXIL::OpCode::VectorReduceOr = 310
```

Comment:
Is is sufficient to reserve the OpCode just in text here, then I'll update the actual enum once the proposal change is accepted?